pages/user-interfaces.html

```diff
@@ -55,102 +43,121 @@
 </p>
 
 <p>
-An experimental alternative to the editors mentioned above is the
-computational notebook interface provided by Jupyter.
-A <a href="https://github.com/EugeneLoy/coq_jupyter/">Jupyter kernel
-for Coq</a> is available if you want to try this interface, for
-instance for pedagogical or readability purposes.  One of the
-advantages of using a Jupyter notebook is that you can save and
-display a selection of intermediate proof steps, which your reader
-will see without the need to reexecute the document.
+Otherwise, <a href="https://coq.inria.fr/refman/practical-tools/coqide.html">CoqIDE</a> is
+a <strong>standalone</strong> interface
+developed and distributed alongside Coq.
+</p>
+
+<p>
+As a way to try Coq without installing anything, you can also
+use <a href="https://jscoq.github.io/">JsCoq</a> which loads Coq
+entirely in <strong>your browser</strong>.  However, it is limited to conduct
+actual projects: it lacks access to the VM and native computing
```

Comment:
Not sure how "it is limited to conduct actual projects" sounds. Depending on your project it can work very well or very bad. I'd rephrase this entirely.

Reply:
OK, but then please suggest an alternative wording. Otherwise, you are just blocking the PR without helping make progress.

Reply:
Note that indeed I'd like we don't block this PR for too long, however I think I see some critical issues with it so I'd be great if we could (as requested below) maybe have a quick visio chat and converge.

Reply:
I can make myself available tomorrow afternoon if needed. Ping me by Zulip or mail.